tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/ListingsSpec.scala

```diff
@@ -258,6 +272,31 @@ final class ListingsSpec extends BaseSpec with Inspectors with EitherValuable wi
       }
     }
 
+    "get the latest revision of previously tagged resources when queried by tag" in {
```

Comment:
This searches on tags used on resources in different projects, is searching across different orgs also worth testing?

Reply:
IMO should be ok since the scope of the search is already tested